alws/crud/build.py

```diff
@@ -40,7 +41,7 @@ async def create_build(
     if not user:
         raise ValueError(f'Cannot find user with id {user_id}')
 
-    if not can_perform(product, user, 'create_build'):
+    if not can_perform(product, user, actions.CreateBuild.name):
```

Comment:
Not critical, and I do know that `can_perform` is already in the codebase, but to me, I'd prefer to read `can_perform(action, over-an-object, as user)`

Reply:
The approach was inspired by casbin examples, like "object subject action"